src/components/Navbar.css

```diff
@@ -0,0 +1,187 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+/* body {
+    margin: -2px;
+    font-family: 'Inter', Tahoma, sans-serif;
+    font-weight : 300;
+    background: linear-gradient(to bottom, #282c34  5%, #07080A 100%);
+} */
+
+/* .content{
+    height: 200vh;
+    color: white;
+    background: linear-gradient(to bottom, #282c34  5%, #07080A 100%);
+    display: grid;
+    place-items: center;
+
+} */
+
+/* Nav features */
+/* @media (m-width: 0px) and (max-width: 700px){ */
+
+.navbar {
+  /* background:#07080A; */
+  background: #07080a;
+  text-align: center;
+  position: fixed;
+  z-index: 999;
+  width: 100%;
+  height: 83.6px;
+}
+
+.navbar__logo{
+  margin-top: 8px;
+  width: 150px;
+
+}
+.navbar__nav-toggle {
+  position: absolute !important;
```

Comment:
Here, this `!important` rule overrides any other declarations which is commonly bad practice because it makes debugging more difficult by breaking the natural cascading in your stylesheets. I would suggest to avoid using this.